### ISSUE TITLE
Any plans to make some docs available?

### ISSUE THREAD

**Issue by ryanford** [vaughantay/roguelove#1]
Would love to give it a try, especially with 7drl coming. Any plans to make some docs or implement ldoc or something?

**Comment by vaughantay** [comment]
It's incomplete, and missing info on features that have been added to the engine that weren't part of Possession, but there's a Possession modding guide that describes how to create various types of content:
https://docs.google.com/document/d/1bJmuokbK8Xtd2P9K8KRQRSeGdHd78HGKuOKaZltCoE4/edit

I actyakky started working on setting up ldoc-style documentation for most of the base classes, so I'll get working on making sure that's cleaned up and available.

**Comment by vaughantay** [comment]
So I've created a new document based on the Possession modding guide where I'm working on stripping out the modding-specific info and going into more detail about the engine itself. It's at https://docs.google.com/document/d/1bJmuokbK8Xtd2P9K8KRQRSeGdHd78HGKuOKaZltCoE4/edit

I've also started documenting some of the classes and other code with ldoc, which is available at:
http://weirdfellows.com/files/roguelove/doc/

Both are very in progress (although anything on the ldoc page should be complete). Unfortunately, I don't know how likely it is to get things in a great state of documentation in time for 7DRL, but I'll be continuing to work on it.
